sources/packages/forms/Dockerfile.dev

```diff
@@ -8,8 +8,8 @@ ARG FORMIO_SOURCE_REPO_TAG
 # Note: using pinned versions to ensure immutable build environment.
 RUN apk update && \
   apk upgrade && \
-  apk add make=4.4.1-r2 && \
-  apk add python3=3.12.6-r0 && \
+  apk add make && \
```

Comment:
Thanks for taking care of this. I would suggest removing the version from the other packages also (lines 13 and 14).